packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1662,6 +1683,20 @@ export default class NewVm extends BaseComponent {
     const { isAdmin } = this.props
     const { formatMessage } = this.props.intl
     const isHvm = this._isHvm()
+    const _copyHostBiosStrings =
+      isAdmin && isHvm ? (
+        <label>
+          <input
+            checked={copyHostBiosStrings}
```

Comment:
[Will it contain them](https://github.com/vatesfr/xen-orchestra/pull/4755#discussion_r381931630)?

Reply:
Yes